app/services/map_marker_json.rb

```diff
@@ -1,9 +1,13 @@
 module MapMarkerJson
-  def self.build(organisations)
-    Gmaps4rails.build_markers(Queries::Organisations.add_recently_updated_and_has_owner(organisations)) do |org, marker|
+  def self.build(organisations, include_extra_organisation_data = true)
+    if include_extra_organisation_data
+      organisations = Queries::Organisations.add_recently_updated_and_has_owner(organisations)
```

Comment:
Metrics/LineLength: Line is too long. [94/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)